docs/source/features/reasoning_outputs.md

```diff
@@ -4,14 +4,16 @@
 
 vLLM offers support for reasoning models like [DeepSeek R1](https://huggingface.co/deepseek-ai/DeepSeek-R1), which are designed to generate outputs containing both reasoning steps and final conclusions.
 
-Reasoning models return a additional `reasoning_content` field in their outputs, which contains the reasoning steps that led to the final conclusion. This field is not present in the outputs of other models.
+Reasoning models return an additional `reasoning_content` field in their outputs, which contains the reasoning steps that led to the final conclusion. This field is not present in the outputs of other models.
 
 ## Supported Models
 
 vLLM currently supports the following reasoning models:
 
 - [DeepSeek R1 series](https://huggingface.co/collections/deepseek-ai/deepseek-r1-678e1e131c0169c0bc89728d) (`deepseek_r1`, which looks for `<think> ... </think>`)
 
+- [IBM Granite 3.2 language models](https://huggingface.co/collections/ibm-granite/granite-32-language-models-67b3bc8c13508f6d064cff9a), which look for `Here is my thought process: ... Here is my response: ...`. Note that for granite models, you must also pass `thinking=True` in your `chat_template_kwargs`.
```

Comment:
Could we mention the parser name here? Maybe like this

```suggestion
- [IBM Granite 3.2 language models](https://huggingface.co/collections/ibm-granite/granite-32-language-models-67b3bc8c13508f6d064cff9a), `granite`, which looks for `Here is my thought process: ... Here is my response: ...`. Note that for granite models, you must also pass `thinking=True` in your `chat_template_kwargs`.
```

Reply:
Sounds good to me!